Utils/release/startMage.sh

```diff
@@ -1,5 +1,13 @@
-#!/bin/sh
-cd ./mage-client
-./startClient.sh
-cd ../mage-server
-./startServer.sh
+#!/bin/sh -x
+
+script_dir="$(readlink -f -- "$BASH_SOURCE")"
+script_dir="$(dirname -- $script_dir)"
+cd $script_dir
+
+# i dont really recommend using this script
+# because both "child" scripts will output
+# to the same terminal
+# instead execute each individually
+
+"${script_dir}/startClient.sh" &
+"${script_dir}/startServer.sh" &
```

Comment:
Both scripts stores in different folder, but you run it from one. It's important to run server and client from mage-server and mage-client folders.

`script_dir="$(dirname -- $script_dir)"`

As I can see it run scripts from parent folder. That's wrong.